Makefile

```diff
@@ -9,10 +9,7 @@ test:
 	ginkgo ./pkg/...
 
 dep:
-	dep ensure -v
+	go get -t -u ./...
```

Comment:
Shouldn't we use `go mod tidy`? That is at least what I am used to from other projects. What is the motivation to use go get here?

Reply:
The way I understood your previous comment was to have a target that would upgrade dependencies to the latest versions and then vendor them in. The ```go get``` upgrades all dependencies to the latest versions. I guess ```go mod tidy```could be added in between of those to make sure the useless ones will get removed after the upgrades. Should we go down this path?

Reply:
I believe that `go mod tidy` alone would try to update unpinned dependencies and clean up, so `go get` would be redundant in that case. Would it make sense?

Reply:
I do not think that ```go mod tidy``` upgrades the dependencies. I just double checked and could not find any info on that. Can you please point me to it? Here is a link to go FAQ on upgrading dependencies: https://github.com/golang/go/wiki/Modules#how-to-upgrade-and-downgrade-dependencies.

Reply:
Oh, I was wrong then. Let's then add go mod tidy for cleanup after the install step and call it a day.

Reply:
No problem, I just added the tidy after install and pushed the branch.